libs/core/executors/include/hpx/executors/datapar/execution_policy.hpp

```diff
@@ -290,8 +293,11 @@ namespace hpx { namespace execution { inline namespace v1 {
 
         using base_policy_type =
             sequenced_task_policy_shim<Executor, Parameters>;
-        static constexpr base_policy_type base_policy =
-            sequenced_task_policy_shim<Executor, Parameters>{};
+        base_policy_type base_policy()
+        {
+            return simd_task_policy_shim<Executor, Parameters>(
+                HPX_FORWARD(Executor, exec_), HPX_FORWARD(Parameters, params_));
+        }
```

Comment:
```suggestion
        base_policy_type base_policy()
        {
            return base_policy_type(exec_, params_);
        }
```
We should not forward the members to the constructor of the new instance, those are not forwarding references,

Reply:
Oops!, missed it completely. Will resolve it ASAP!,
Thanks! :)